servers/src/main/java/tachyon/master/journal/JournalProtoUtils.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.master.journal;
+
+import com.google.protobuf.Message;
+
+import tachyon.exception.ExceptionMessage;
+import tachyon.proto.JournalEntryProtos.JournalEntry;
+
+/**
+ * Utils for working with the journal.
+ */
+public final class JournalProtoUtils {
```

Comment:
+1 for this ... it might make sense to call these methods `encode` and `decode` or `wrap` and `unwrap` or `getMessage` and `setMessage` ... `getInnerEntry` introduces a new term that does not seem to be needed